Presentation.Web/Controllers/API/ItSystemUsageDataSensitivityLevelController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.ComponentModel;
+using System.Net.Http;
+using System.Web.Http;
+using Core.ApplicationServices.SystemUsage;
+using Core.DomainModel.ItSystemUsage.GDPR;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models.ItSystemUsage;
+
+namespace Presentation.Web.Controllers.API
+{
+    [PublicApi]
+    [RoutePrefix("api/v1/itsystemusage")]
+    public class ItSystemUsageDataSensitivityLevelController : BaseApiController
+    {
+
+        private readonly IItSystemUsageService _usageService;
+
+        public ItSystemUsageDataSensitivityLevelController(IItSystemUsageService usageService)
+        {
+            _usageService = usageService;
+        }
+
+        /// <summary>
+        /// Opretter en ny systemrelation
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="dataSensitivityLevel"></param>
+        /// <returns></returns>
+        [HttpPatch]
+        [Route("{id}/sensitivityLevel/add/{dataSensitivityLevel}")]
+        public HttpResponseMessage AddSensitivityLevel(int id, int dataSensitivityLevel)
+        {
+            var sensitivityLevel = MapSensitivityLevel(dataSensitivityLevel);
+            var result = _usageService.AddSensitiveDataLevel(id, sensitivityLevel);
+
+            return result.Match
+            (
+                onSuccess: itSystemUsageDataLevel =>  Ok(Map<ItSystemUsageSensitiveDataLevel,ItSystemUsageSensitiveDataLevelDTO>(itSystemUsageDataLevel)),
+                onFailure: FromOperationError
+            );
+        }
+
+        [HttpPatch]
+        [Route("{id}/sensitivityLevel/remove/{dataSensitivityLevel}")]
+        public HttpResponseMessage RemoveSensitivityLevel(int id, int dataSensitivityLevel)
+        {
+            var sensitivityLevel = MapSensitivityLevel(dataSensitivityLevel);
```

Comment:
Hvis du tager en patch body i stedet for eller en model med FromUri på attributes så kan du få den ud som enum i stedet og slippe for det her